docs/detections/add-exceptions.asciidoc

```diff
@@ -0,0 +1,241 @@
+
+. Select one of the following alert actions:
+
+* *Close this alert*: Closes the alert when the exception is added. This option
+is only available when adding exceptions from the Alerts table.
+* *Close all alerts that match this exception and were generated by this rule*:
+Closes all alerts that match the exception's conditions and were generated only by the current rule.
++
+. Click *Add rule exception*. 
+
+[float]
+[[endpoint-rule-exceptions]]
+=== Add {elastic-endpoint} exceptions
+
+Like detection rule exceptions, you can add Endpoint agent exceptions either by editing the Endpoint Security rule or by adding them as actions on alerts generated by the Endpoint Security rule. {elastic-endpoint} alerts have the following fields:
+
+* `kibana.alert.original_event.module determined:endpoint`
+* `kibana.alert.original_event.kind:alert`
+
+You can also add Endpoint exceptions to rules that are associated with {elastic-endpoint} rule exceptions. To associate rules, when creating or editing a rule, select the <<rule-ui-advanced-params, *{elastic-endpoint} exceptions*>> option.
+
+Endpoint exceptions are added to
+both the detection rule *and* the {elastic-endpoint} agent on your hosts.
+
+[IMPORTANT]
+=============
+Exceptions added to the Endpoint Security rule affect all alerts sent
+from the Endpoint agent. Be careful not to unintentionally prevent some Endpoint
+alerts.
+
+Additionally, to add an Endpoint exception to the Endpoint Security rule, there must be at least one Endpoint Security alert generated in the system. For non-production use, if no alerts exist, you can trigger a test alert using malware emulation techniques or tools such as the Anti Malware Testfile from the https://www.eicar.org/[European Institute for Computer Anti-Virus Research (EICAR)].
+=============
+
+[IMPORTANT]
+=====
+{ref}/binary.html[Binary fields] are not supported in detection rule exceptions.
+=====
+
+. Do one of the following:
++
+--
+
+* To add an Endpoint exception from the rule details page:
+.. Go to the rule details page (*Manage* -> *Rules*), and then search for and select the Elastic *Endpoint Security* rule.
+.. Scroll down the rule details page, select the *Endpoint exceptions* tab, then click *Add endpoint exception*.
+
+* To add an Endpoint exception from the Alerts table:
+.. Go to *Alerts*.
+.. Scroll down to the Alerts table, and from an {elastic-endpoint}
+alert, click the *More actions* menu (*...*), then select *Add Endpoint exception*.
+
+* To add an Endpoint exception from Rule Exceptions page:
+.. Go to *Manage* -> *Rule Exceptions*.
+.. Expand the Endpoint Security Exception List or click the list name to open the list's details page. Next, click *Add endpoint exception*. 
++
+NOTE: The Endpoint Security Exception List is automatically created. By default, it's associated with the Endpoint Security rule and with any rules that have the <<rule-ui-advanced-params, *{elastic-endpoint} exceptions*>> option selected.  
```

Comment:
```suggestion
NOTE: The Endpoint Security Exception List is automatically created. By default, it's associated with the Endpoint Security rule any rules with the <<rule-ui-advanced-params, *{elastic-endpoint} exceptions*>> option selected.  
```

Reply:
```suggestion
NOTE: The Endpoint Security Exception List is automatically created. By default, it's associated with the Endpoint Security rule and any rules with the <<rule-ui-advanced-params, *{elastic-endpoint} exceptions*>> option selected.
```

Reply:
Re-added "and" because I think it was accidentally removed.